joblib/test/test_parallel.py

```diff
@@ -286,17 +284,15 @@ def g(x):
 def test_parallel_timeout_success():
     # Check that timeout isn't thrown when function is fast enough
     for backend in ['multiprocessing', 'threading']:
-        nose.tools.assert_equal(
-            10,
-            len(Parallel(n_jobs=2, backend=backend, timeout=10)
-                (delayed(sleep)(0.001) for x in range(10))))
+        assert_equal(10, len(Parallel(n_jobs=2, backend=backend, timeout=10)
```

Comment:
for readability, I would split the lines here: one for the expected value, one (well 2) for the tested expression.